tests/unit/store/test_version_store.py

```diff
@@ -238,6 +238,22 @@ def test_snapshot():
                                                 call('foo', as_of=None, read_preference=ReadPreference.PRIMARY)])
 
 
+def test_list_symbols_default_pipeline():
+    versions = Mock()
+    vs = create_autospec(VersionStore, _versions=versions)
+    versions.aggregate.return_value = []
+
+    VersionStore.list_symbols(vs)
```

Comment:
whats the purpose of this? this returns a list right? it doesnt seem like the return value is being captured or used.

Reply:
Looks like the unit test is just asserting that pymongo aggregation pipeline is being called correctly.  For this mocked case the `list_symbols` will return the `return_value` from above.